City-of-Helsinki/drupal-infofinland
# IFU-309:  theme set up

Added Infofinland theme.

To test:

- Run `make new`
- Check that Infofinland theme is in use
- Try running `npm i`, `npm run dev`, `npm run build` inside theme to confirm these work on all machines
- Check code, is there anything to add / sketchy / confusing? Should something else be added from the HDBT theme to this one?
